ui/packages/catalog-ui-search/src/main/webapp/component/workspace-interactions/workspace-interactions.view.js

```diff
@@ -66,10 +66,23 @@ define([
     onRender: function() {
       this.checkIfSubscribed()
       this.handleLocal()
+      this.handleShareable()
     },
     handleLocal: function() {
       this.$el.toggleClass('is-local', this.model.isLocal())
     },
+    handleShareable: function() {
+      const notShareable = function(that) {
+        const userLogin = user.get('user').get('email')
```

Comment:
❓ Would there ever be a case where user is null/undefined?

Reply:
good question, I saw `user` used everywhere without any checks so I assumed it was safe.

For good measure, I just searched the code for `user.*undefined` and `user.*null` and didn't see any place where we check before using `user`. https://github.com/codice/ddf/blob/1028a346615fccb33ac60125dac753c6a9e09cac/ui/packages/catalog-ui-search/src/main/webapp/js/model/User.js#L49

Reply:
it just occurred to me that you may be talking about the return from `user.get('user')`, I'm looking into that now

Reply:
looks safe to me